llama_index/indices/service_context.py

```diff
@@ -115,12 +129,10 @@ def from_defaults(
         embed_model = embed_model or OpenAIEmbedding()
         embed_model.callback_manager = callback_manager
 
-        prompt_helper = prompt_helper or PromptHelper.from_llm_predictor(
-            llm_predictor, chunk_size_limit=chunk_size_limit
-        )
+        prompt_helper = prompt_helper or PromptHelper.from_llm_predictor(llm_predictor)
 
         node_parser = node_parser or _get_default_node_parser(
-            chunk_size_limit=chunk_size_limit, callback_manager=callback_manager
+            chunk_size=chunk_size, callback_manager=callback_manager
```

Comment:
I think jerry mentioned this in passing, but I think we also to be able to propagate chunk overlap to the node parser as well

Reply:
done